Examples/Nearest_Neighbour_Distance.ipynb

```diff
@@ -31,7 +31,9 @@
    },
```

Comment:
<p>I suggest getting rid of the cat subset line and merging into the var_search line.</p><p> </p><p>and changing <code>darray</code> to <code>ds</code> because its a dataset not a data array</p><p> </p><p>This would be similar to cell 4 in <a href="https://github.com/COSIMA/cosima-recipes/blob/4fd51150331ebb1669fdb8bc58ff4ac16b313b0e/Examples/Bathymetry.ipynb" target="_blank">https://github.com/COSIMA/cosima-recipes/blob/4fd51150331ebb1669fdb8bc58ff4ac16b313b0e/Examples/Bathymetry.ipynb</a>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>